superset-frontend/src/SqlLab/types.ts

```diff
@@ -66,33 +66,19 @@ export type SqlLabRootState = {
     tabHistory: string[]; // default is activeTab ? [activeTab.id.toString()] : []
     tables: Record<string, any>[];
     queriesLastUpdate: number;
-    user: UserWithPermissionsAndRoles;
     errorMessage: string | null;
     unsavedQueryEditor: Partial<QueryEditor>;
     queryCostEstimates?: Record<string, QueryCostEstimate>;
   };
   localStorageUsageInKilobytes: number;
   messageToasts: toastState[];
+  user: UserWithPermissionsAndRoles;
   common: {
     flash_messages: string[];
     conf: JsonObject;
   };
 };
 
-export type SqlLabExploreRootState = SqlLabRootState | RootState;
```

Comment:
@lyndsiWilliams I moved [this user selector logic](https://github.com/apache/superset/pull/19855/files#diff-98bfde8ed111b1aa1a94f5da25ed87585b19908a628063fd133958e9576c239aR69-R82) since it's under the same path.